classes/User.php

```diff
@@ -254,6 +254,7 @@ public function loadData($UPDATE = 2, $remote_view = false): string {
 		}
 		else {
 			$blacklist_json = json_encode($this->blacklist);
+			$this->original_blacklist = $this->blacklist;
```

Comment:
let's revert this, fix merged in https://github.com/levimeahan/shinobi-chronicles/pull/106